sdks/python/scripts/run_pylint.sh

```diff
@@ -37,6 +37,11 @@ fi
 set -o errexit
 set -o pipefail
 
+# Check for trailing spaces in non-Python text files.
+echo "Running whitespacelint..."
+MARKDOWN=$(find apache_beam -name '*.md' -print0)
```

Comment:
a bit strange to apply this only when there is a python change. md files could be anywhere in the repo.

Reply:
Yes, it makes sense to put it at the root.

I've made a new task to run the tool in a python virtual env, but for all files.
It'll be triggered if there are markdown files or build.gradle files changed.